### PR TITLE
snip TSB links

### PR DESCRIPTION
**Pull request by deads2k**
This removes a couple unnecessary dependencies from the TSB in preparation for a later switch to https://github.com/openshift/client-go

### CODE REVIEW

File: pkg/serviceaccounts/client.go [hunk 1]
```diff
@@ -1,119 +1,11 @@
...
-	"fmt"
-	"time"
-
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	restclient "k8s.io/client-go/rest"
 	kapi "k8s.io/kubernetes/pkg/api"
-	kclientsetexternal "k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
-	kclientsetinternal "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
-
-	"github.com/openshift/origin/pkg/client"
```
Comment (deads2k) [inline pkg/serviceaccounts/client.go]: @stevekuznetsov this transitive wasn't picked up before.  Weird.  Known?

Comment (stevekuznetsov) [reply pkg/serviceaccounts/client.go]: Did we merge upstream fix and rebase to grab it? There was an issue with prefixing.

Comment (deads2k) [reply pkg/serviceaccounts/client.go]: > Did we merge upstream fix and rebase to grab it? There was an issue with prefixing.

Ok, known.  Thanks.

**Pull request merged**